.github/workflows/ci_push.yml

```
@@ -112,6 +112,8 @@ jobs:
       run: |
         .github/workflows/install_deps.sh ipopt
         pip install -e .[test,ipopt,dlib,pyswarm,cmaes,nlopt,fides]
```

```suggestion
        pip install -e .[test,ipopt,dlib,pyswarm,cmaes,nlopt,fides,MPI]
```

as you added it to setup.py already

---

for consistency, maybe lower-case?